src/test/java/PlaylistTests.java

```diff
@@ -0,0 +1,56 @@
+import net.bytebuddy.asm.Advice;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaylistTests extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://github.com/Test-Pro-Bootcamp/A44-QA-Automation-Framework/assets/104761482/a8906508-128e-4195-8e19-9fc52d369c6c)